test/mocks/agolItems.ts

```diff
@@ -714,7 +724,13 @@ export function createAGOLRelationship (
 export function getAnImageResponse (
 ): any {
   const fs = require("fs");
-  return fs.createReadStream("./test/mocks/success.png");
+  if (fs.createReadStream) {
+    // Node test
+    return fs.createReadStream("./test/mocks/success.png");
+  } else {
+    // Chrome test
+    return new Blob([atob(imageAsDataUri(false))], {type: "image/png"})
```

Comment:
minor issue but missing a `;`

Reply:
Thank you; I probably should add linting to the tests